app/moves/middleware/redirect-default-query.js

```diff
@@ -0,0 +1,16 @@
+const { isEmpty } = require('lodash')
+const querystring = require('qs')
+
+function redirectDefaultQuery(defaults = {}) {
+  return function handleQueryRedirect(req, res, next, view) {
+    const query = defaults[view]
+
+    if (!isEmpty(query) && isEmpty(req.query)) {
+      return res.redirect(`${req.originalUrl}?${querystring.stringify(query)}`)
```

Comment:
Personally I'd prefer to have the url construction in another method. a) it makes testing both the construction and the redirection simpler and  b) it could be reused.

Reply:
I'm inclined to agree for a more complex URL, which there will be some coming but this one felt simple enough to leave within this method at this stage.